[2025-08-16T18:41:27.525+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-16T18:41:27.601+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_athena_pipeline.dbt_stage_external_sources manual__2025-08-16T18:24:26+00:00 [queued]>
[2025-08-16T18:41:27.622+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_athena_pipeline.dbt_stage_external_sources manual__2025-08-16T18:24:26+00:00 [queued]>
[2025-08-16T18:41:27.623+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-08-16T18:41:27.666+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_stage_external_sources> on 2025-08-16 18:24:26+00:00
[2025-08-16T18:41:27.688+0000] {standard_task_runner.py:64} INFO - Started process 141 to run task
[2025-08-16T18:41:27.700+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_athena_pipeline', 'dbt_stage_external_sources', 'manual__2025-08-16T18:24:26+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dbt_athena_pipeline.py', '--cfg-path', '/tmp/tmpz2o4o1tf']
[2025-08-16T18:41:27.704+0000] {standard_task_runner.py:91} INFO - Job 41: Subtask dbt_stage_external_sources
[2025-08-16T18:41:27.884+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_athena_pipeline.dbt_stage_external_sources manual__2025-08-16T18:24:26+00:00 [running]> on host 392a94efd1c1
[2025-08-16T18:41:28.193+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_athena_pipeline' AIRFLOW_CTX_TASK_ID='dbt_stage_external_sources' AIRFLOW_CTX_EXECUTION_DATE='2025-08-16T18:24:26+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-16T18:24:26+00:00'
[2025-08-16T18:41:28.196+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-16T18:41:28.236+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-16T18:41:28.239+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -Eeuo pipefail -x; cd /opt/***/dbt; echo \'which dbt=\'$(command -v dbt || true); /home/***/.local/bin/dbt --version || true; echo "ENV S3_BUCKET=$S3_BUCKET ATHENA_DB=$ATHENA_DB AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION"; /home/***/.local/bin/dbt run-operation stage_external_sources --profiles-dir /opt/***/.dbt --target dev -v']
[2025-08-16T18:41:28.266+0000] {subprocess.py:86} INFO - Output:
[2025-08-16T18:41:28.269+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-08-16T18:41:28.270+0000] {subprocess.py:93} INFO - ++ command -v dbt
[2025-08-16T18:41:28.271+0000] {subprocess.py:93} INFO - + echo 'which dbt=/home/***/.local/bin/dbt'
[2025-08-16T18:41:28.272+0000] {subprocess.py:93} INFO - which dbt=/home/***/.local/bin/dbt
[2025-08-16T18:41:28.273+0000] {subprocess.py:93} INFO - + /home/***/.local/bin/dbt --version
[2025-08-16T18:41:36.775+0000] {subprocess.py:93} INFO - Core:
[2025-08-16T18:41:36.776+0000] {subprocess.py:93} INFO -   - installed: 1.9.2
[2025-08-16T18:41:36.777+0000] {subprocess.py:93} INFO -   - latest:    1.10.9 - Update available!
[2025-08-16T18:41:36.777+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:36.778+0000] {subprocess.py:93} INFO -   Your version of dbt-core is out of date!
[2025-08-16T18:41:36.778+0000] {subprocess.py:93} INFO -   You can find instructions for upgrading here:
[2025-08-16T18:41:36.779+0000] {subprocess.py:93} INFO -   https://docs.getdbt.com/docs/installation
[2025-08-16T18:41:36.779+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:36.780+0000] {subprocess.py:93} INFO - Plugins:
[2025-08-16T18:41:36.781+0000] {subprocess.py:93} INFO -   - athena: 1.9.4 - Up to date!
[2025-08-16T18:41:36.781+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:36.782+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:37.384+0000] {subprocess.py:93} INFO - + echo 'ENV S3_BUCKET=pt-data-lab ATHENA_DB=analytics AWS_DEFAULT_REGION=us-east-2'
[2025-08-16T18:41:37.384+0000] {subprocess.py:93} INFO - ENV S3_BUCKET=pt-data-lab ATHENA_DB=analytics AWS_DEFAULT_REGION=us-east-2
[2025-08-16T18:41:37.385+0000] {subprocess.py:93} INFO - + /home/***/.local/bin/dbt run-operation stage_external_sources --profiles-dir /opt/***/.dbt --target dev -v
[2025-08-16T18:41:46.455+0000] {subprocess.py:93} INFO - Core:
[2025-08-16T18:41:46.457+0000] {subprocess.py:93} INFO -   - installed: 1.9.2
[2025-08-16T18:41:46.459+0000] {subprocess.py:93} INFO -   - latest:    1.10.9 - Update available!
[2025-08-16T18:41:46.461+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:46.462+0000] {subprocess.py:93} INFO -   Your version of dbt-core is out of date!
[2025-08-16T18:41:46.463+0000] {subprocess.py:93} INFO -   You can find instructions for upgrading here:
[2025-08-16T18:41:46.464+0000] {subprocess.py:93} INFO -   https://docs.getdbt.com/docs/installation
[2025-08-16T18:41:46.464+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:46.465+0000] {subprocess.py:93} INFO - Plugins:
[2025-08-16T18:41:46.466+0000] {subprocess.py:93} INFO -   - athena: 1.9.4 - Up to date!
[2025-08-16T18:41:46.466+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:46.467+0000] {subprocess.py:93} INFO - 
[2025-08-16T18:41:47.154+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-16T18:41:47.156+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-16T18:41:47.208+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_athena_pipeline, task_id=dbt_stage_external_sources, run_id=manual__2025-08-16T18:24:26+00:00, execution_date=20250816T182426, start_date=20250816T184127, end_date=20250816T184147
[2025-08-16T18:41:47.284+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-16T18:41:47.346+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-16T18:41:47.350+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
